tools/pip-requires

```diff
@@ -19,3 +19,5 @@ numpy==1.14.2                                     # Numpy for forecasting T3C
 paramiko==2.6.0                                   # SSH2 protocol library
 Flask==1.1.1                                      # Python web framework
 M2Crypto<=0.35.2                                  # Dependency of FTS rest API; Temporary fix since 0.33 does not install on CC7
+MyProxyClient==2.1.0                              # myproxy client bindings
```

Comment:
this is gone, right?

Reply:
This must have been added during a rebase ... I removed the line 22 now.